lib/engine.js

```diff
@@ -25,9 +25,13 @@ let towerButtons = $('.tower-button');
 let startButton = document.getElementById('start');
 let numberOfLives = document.getElementById('number-of-lives');
 let moneyAmount = document.getElementById('money-amount');
-
 let imageLoader = new ImageLoader();
-let images = imageLoader.init();
+let images = imageLoader.init(game.currentLevel.stage);
+
+function loadImages() {
+  imageLoader = new ImageLoader();
+  images = imageLoader.init(game.currentLevel.stage);
+}
```

Comment:
Is there a reason why these two lines are both in a function and outside of the function?